sdk/identity/identity/CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ This release continues with the changes from `1.2.4` and `1.2.4-beta.1`.
 - Breaking Change: `InteractiveBrowserCredential` for the browser now requires the client ID to be provided.
 - Documentation was added to elaborate on how to configure an AAD application to support `InteractiveBrowserCredential`.
 - Replaced the use of the 'express' module with a Node-native http server, shrinking the resulting identity module considerably
+- Updated `@azure/msal-node-extensions` to [1.0.0-alpha.6](https://www.npmjs.com/package/@azure/msal-node-extensions/v/1.0.0-alpha.6).
+- Refactored our use of MSAL to better centralize the handling of inputs, outputs and errors.
+- Migrated the `InteractiveBrowserCredential`, `DeviceCodeCredential`, `ClientSecretCredential`, `ClientCertificateCredential`, `EnvironmentCredential` and `UsernamePasswordCredential` to the latest MSAL.
+  - This update improves caching of tokens, significantly reducing the number of network requests.
+- Credentials `InteractiveBrowserCredential`, `DeviceCodeCredential`, `EnvironmentCredential` and `UsernamePasswordCredential` now can:
```

Comment:
The changes listed here don't seem to apply to `EnvironmentCredential` (which they should not), should it be removed from here?

Reply:
That is true! I'll update the changelog.

Reply:
Removed the EnvironmentCredential mention. Thank you for spotting this!